tests/Fixer/LanguageConstruct/FunctionToConstantFixerTest.php

```diff
@@ -126,6 +126,30 @@ public function provideTestCases()
                 '<?php echo phpversion(); echo php_sapi_name(); echo M_PI;',
                 ['functions' => ['pi', 'phpversion']],
             ],
+            'get_class => T_CLASS' => [
```

Comment:
TBH I think it is an odd thing that this wasn't patched in PHP to act the same when they added the warning.
Not sure checking the calling `get_class` outside class `""` vs. `false` is worth the effort. The reason for that is that it makes no sense anyway to do it, as all it is is a very expensive way to get `false` or `''`.